pkg/controllers/dynakube/logmonitoring/daemonset/annotations.go

```diff
@@ -2,21 +2,11 @@ package daemonset
 
 import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/api"
-	maputils "github.com/Dynatrace/dynatrace-operator/pkg/util/map"
```

Comment:
out of curiosity can we replace it with default maps? (unrelated to this PR)

Reply:
there is no `Merge` in the default `maps` lib, AFAIK